templates/app/webpack-default/template/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+	"recommendations": [
+		"axway.vscode-titanium"
```

Comment:
Love it

Reply:
I didn't know we can do this. We should do the same in all of our templates.

Reply:
@janvennemann, @ewanharris, maybe we should add a `/.vscode/settings.json` file which tells the editor to ignore our generated directories. What do you think?
```json
{
	"files.exclude": {
		"build/": true,
		"i18n/": true,
		"platform/": true,
		"Resources/": true
	},
	"search.exclude": {
		"build/": true,
		"i18n/": true,
		"platform/": true,
		"Resources/": true
	},
	"files.watcherExclude": {
		"build/": true,
		"i18n/": true,
		"platform/": true,
		"Resources/": true
	}
}
```

Reply:
I wrote up a ticket to add these vscode files to our other app templates here...
https://jira.appcelerator.org/browse/TIMOB-28070